cmds/import_image/import_image_cmd.go

```diff
@@ -4,20 +4,46 @@ import (
 	"errors"
 	"fmt"
 
-	common "github.com/cloudfoundry-community/bosh-softlayer-tools/common"
+	"github.com/cloudfoundry-community/bosh-softlayer-tools/common"
 
+	boshlog "github.com/cloudfoundry/bosh-utils/logger"
+	boshretry "github.com/cloudfoundry/bosh-utils/retrystrategy"
 	sldatatypes "github.com/maximilien/softlayer-go/data_types"
-	softlayer "github.com/maximilien/softlayer-go/softlayer"
+	"github.com/maximilien/softlayer-go/softlayer"
+	"github.com/pivotal-golang/clock"
+)
+
+var (
```

Comment:
Do what? you want to publish the image to all DCs? We already tested that we only published the image into lon02 DC, and we tried to deploy a VM with this image in dal09 DC, it's successful, so I don't think it's necessary to populate the image to all DCs.